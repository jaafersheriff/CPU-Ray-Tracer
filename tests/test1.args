pixelray simple.pov 640 480 320 240
