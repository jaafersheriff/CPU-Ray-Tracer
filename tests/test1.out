Pixel: [320, 240] Ray: {0 0 14} -> {0.001042 0.001042 -1}
